tests_proc_dict.py

```diff
@@ -58,6 +60,47 @@ def continuous_annotation():
     }
 
 
+@pytest.fixture
+def missing_value():
```

Comment:
Is this fixture needed? Unless I'm missing something, I don't see it used anywhere

Reply:
I think you are right.